docs/CONFIGURATION.md

```diff
@@ -217,7 +217,8 @@ The agent table configures Telegraf and the defaults used across all plugins.
 
 - **logfile**:
   Name of the file to be logged to when using the "file" logtarget.  If set to
-  the empty string then logs are written to stderr.
+  the empty string then logs are written to stderr. Currently, logs are UTC+0
+  only.
```

Comment:
I would prefer this said, "By default, logs are recorded with the `GMT+00:00 UTC` timezone unless the `log_with_timezone` option is set." The way this reads, setting the timezone with the option below does not work.

Reply:
For the record here is my test using without and then with setting `log_with_timezone`:

```
2022-01-04T15:27:09Z I! Loaded inputs: mem
2022-01-04T15:27:09Z I! Loaded aggregators: 
2022-01-04T15:27:09Z I! Loaded processors: 
2022-01-04T15:27:09Z I! Loaded outputs: file
2022-01-04T15:27:09Z I! Tags enabled: host=ryzen
2022-01-04T15:27:09Z I! [agent] Config: Interval:10s, Quiet:false, Hostname:"ryzen", Flush Interval:10s
2022-01-04T15:27:26Z I! [agent] Hang on, flushing any cached metrics before shutdown
2022-01-04T15:27:26Z I! [agent] Stopping running outputs
2022-01-04T08:27:53-07:00 I! Loaded inputs: mem
2022-01-04T08:27:53-07:00 I! Loaded aggregators: 
2022-01-04T08:27:53-07:00 I! Loaded processors: 
2022-01-04T08:27:53-07:00 I! Loaded outputs: file
2022-01-04T08:27:53-07:00 I! Tags enabled: host=ryzen
2022-01-04T08:27:53-07:00 I! [agent] Config: Interval:10s, Quiet:false, Hostname:"ryzen", Flush Interval:10s
2022-01-04T08:28:15-07:00 I! [agent] Hang on, flushing any cached metrics before shutdown
2022-01-04T08:28:15-07:00 I! [agent] Stopping running outputs
```

Reply:
Sorry i was not aware of the `log_with_timezone` configuration option